### PR TITLE
Cat breed version 1

### PR DESCRIPTION
**Pull request by sabrinatoro**
- added NCBITaxon for cat
- added new AP

### CODE REVIEW

**Comment by sabrinatoro** [comment]
@matentzn 
1) could you please review this PR? thank you!

2) we have a problem with one breed (see line261 [here](https://docs.google.com/spreadsheets/d/19ltiiA66eKjQdNeWaLpaAxE9XoQRJiwXWOcLmugwzXo/edit#gid=0)). @katiermullen and I did a lot of troubleshooting, and it turns out that the breed name "Cyprus" is problematic (we didn't include it in the PR here). Having that breed name gives the following error when we create the catbreeds component: `Index -1 out of bounds for length 36` (note that 36 is the number of columns). I tried changing the name to "somecat", and it solved the problem (ie the name itself is the problem). Do you have any explanation and advice on how to proceed?
Thank you!

File: src/ontology/vbo-edit.owl [hunk 1]
```diff
@@ -48,6 +67,12 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/vbo#existsincount
 AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/vbo#existsincountryiso> "This annotation property refers to the country in which the breed exist, by using the UN ISO standard three-letter country codes. The numeric codes are available from https://www.iso.org/iso-3166-country-codes.html")
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/vbo#existsincountryiso> "exists_in_country_ISOcode")
 
+# Annotation Property: <http://purl.obolibrary.org/obo/vbo#fully_recognized> (fully recognized)
+
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/vbo#fully_recognized> "Recognized breed or championship breed.")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/vbo#fully_recognized> "fully recognized")
+SubAnnotationPropertyOf(<http://purl.obolibrary.org/obo/vbo#fully_recognized> <http://purl.obolibrary.org/obo/vbo#breed_recognition_status_type>)
```
Comment (matentzn) [inline src/ontology/vbo-edit.owl:74]: I suggest to model the recognition status as classes. You can basically leave the templates as they are, but you 

- make a new class "breed recognition status" (replacing breed_recognition_status_type)
- put all the `not_recognized` as proper VBO classes underneath
- leave `http://purl.obolibrary.org/obo/vbo#breed_recognition_status` as is

Comment (sabrinatoro) [reply src/ontology/vbo-edit.owl]: done. I also created classes for other breed status (e.g. domestication status).

**Comment by sabrinatoro** [comment]
This PR addresses #34

**Comment by sabrinatoro** [comment]
@matentzn we made the breed status classes as you suggested. I think it is ready to merge, but I want first to check whether you want to take another look at this PR before I do. Thanks!

File: src/ontology/components/breedstatus.owl [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+        <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">breed status</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/VBO_0300001 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/VBO_0300001">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/VBO_0300000"/>
+        <obo:IAO_0000115 rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Acknowledgment, by an international breed organization, of a group of animal as a breed.</obo:IAO_0000115>
```
Comment (matentzn) [inline src/ontology/components/breedstatus.owl:61]: clarify definition as it sounds like a positive affirmation ("it is acknowledged").

File: src/ontology/components/breedstatus.owl [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+        <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">domestication status</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/VBO_0300006 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/VBO_0300006">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/VBO_0300005"/>
+        <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">feral</rdfs:label>
```
Comment (matentzn) [inline src/ontology/components/breedstatus.owl:138]: Why not be a good citizen and add definitions to all of these :)

Comment (matentzn) [reply src/ontology/components/breedstatus.owl:138]: (no term without definition, unless self explanatory, like chemical or breed)

**Review by matentzn — COMMENTED** [review]
Gotta say, pretty awesome what you are doing here. Very excited. Goood job!

File: src/ontology/components/breedstatus.owl (hunk 3 above)
Comment (sabrinatoro) [reply src/ontology/components/breedstatus.owl:138]: We are working on these definitions (#44 )

File: src/ontology/components/breedstatus.owl (hunk 2 above)
Comment (sabrinatoro) [reply src/ontology/components/breedstatus.owl]: done

**Pull request merged**